domaintools_phisheye/plugin.spec.yaml

```diff
@@ -0,0 +1,129 @@
+plugin_spec_version: v2
+extension: plugin
+products: [insightconnect]
+name: domaintools_phisheye
+title: DomainTools PhishEye
+description: Monitor the Internet for 'Phishy' Domains
+version: 1.0.0
+vendor: rapid7
+support: community
+status: []
+resources:
+  source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/domaintools_phisheye
+  license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
+tags:
+- domaintools
+- phisheye
+hub_tags:
+  use_cases: [data_enrichment, threat_detection_and_response]
+  keywords: [domaintools, phisheye]
+  features: []
+connection:
+  username:
+    type: string
+    description: API username
+    required: true
+    example: user1
+  api_key:
+    title: API Key
+    type: credential_secret_key
+    description: API key e.g. 11111-aaaaa-aaa11-111aa-aaa11
+    required: true
+    example: 11111-aaaaa-aaa11-111aa-aaa11
+types:
+  ip_addresses:
+    ip:
+      title: IPv4
+      description: IPv4 address
+      type: string
+      required: true
+      example: 1.1.1.1
+    country_code:
+      title: Country Code
+      description: Country code
+      type: string
+      required: true
+      example: US
+  domains:
+    domain:
+      title: Domain
+      description: Links to WHOIS page for the domain
+      type: string
+      required: true
+      example: appeltypoexample.com
+    tld:
+      title: TLD
+      description: TLD domain
+      type: string
+      required: true
+      example: com
+    created_date:
+      title: Created Date
+      description: Date when domain created
+      type: string
+      required: false
+      example: 2016-10-31
+    registrant_email:
+      title: Registrant Email
+      description: Email used for register
+      type: string
+      required: false
+      example: user@example.com
+    ip_addresses:
+      title: IP Addresses
+      description: IPv4 addresses
+      type: "[]ip_addresses"
+      required: false
+    name_servers:
+      title: Name Servers
+      description: Name servers used by domain
+      type: "[]string"
+      required: false
+      example: ["ns57.domaincontrol.com", "ns58.domaincontrol.com"]
+    registrar_name:
+      title: Registrar Name
+      description: Registrar name where domain was registered
+      type: string
+      required: false
+      example: GoDaddy.com, LLC
+    risk_score:
+      title: Risk Score
+      description: Calculated by the Domain Risk Score
+      type: integer
+      required: false
+      example: 24
+actions:
+  domain_list:
+    title: Domain List
+    description: Returns domain results for monitored terms
+    input:
+      query:
+        title: Query
+        type: string
+        description: "Term for which the day's domains are desired"
+        required: true
+        example: example
+      days_back:
```

Comment:
@mgorny-morecode can you make this an `enum` of 1-7 since we don't want users passing numbers out of that range.